app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
+    @article = Article.find(params[:article_id])
+    @comment = @article.comments.create(comment_params)
```

Comment:
@sansuaza what happens if for some reason this line fails or returns an error?

Reply:
if this happens, the comment will not be added to the arcicle

Reply:
rails returns the error (undefined method 'comments' for a nill class), so its better to do a validation before trying to add the comment

Reply:
@sansuaza Excellent that is the answer that I was looking for.